exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/LogInLogOutResources.java

```diff
@@ -120,6 +123,12 @@ public Viewable getLoginPageAfterValidationError() {
   public void logout(@Context HttpServletRequest req, @Context HttpServletResponse resp) throws Exception {
     final HttpSession session = req.getSession();
     if (session != null) {
+      final Object authCreds = session.getAttribute(SessionAuthentication.__J_AUTHENTICATED);
+      if (authCreds != null) {
+        final SessionAuthentication sessionAuth = (SessionAuthentication) authCreds;
+        logger.info("WebUser {} logged out from {}:{}", sessionAuth.getUserIdentity().getUserPrincipal().getName(), req
```

Comment:
```suggestion
        logger.info("WebUser {} logged out from {}: {}", sessionAuth.getUserIdentity().getUserPrincipal().getName(), req
```

Reply:
I will prefer it without space since that's the usual convention for `ip:port` string.

Reply:
Oh, I see, than I agree.